modules/metastorage-client/src/main/java/org/apache/ignite/internal/metastorage/client/MetaStorageServiceImpl.java

```diff
@@ -498,7 +499,7 @@ private void stopWatch(IgniteUuid watchId) {
 
                                 v.cursor.close();
                             } catch (InterruptedException e) {
-                                throw new IgniteInternalException(e);
+                                throw new IgniteInternalException(WATCH_STOPPING_ERR, e);
```

Comment:
We may create a MetaStorageException for the module, this would make code easier to read I believe

Reply:
MetaStorageException has been added